packages/@atjson/offset-annotations/src/annotations/section.ts

```diff
@@ -0,0 +1,6 @@
+import { BlockAnnotation } from "@atjson/document";
+
+export default class Section extends BlockAnnotation {
```

Comment:
I guess at some point we'll add decoration style to paragraphs in addition to this??